### DISCUSSION TITLE
GridTemplateColumns doesn't work with FluentDataGrid on release v4.7.0

### DISCUSSION THREAD

**Discussion by michaelc06** [microsoft/fluentui-blazor#1943] [General]
The new release v4.7.0 appears to be causing `GridTemplateColumns` not to work with `FluentDataGrid` and setting all column width to `1fr`. Could be caused by changes in [DataGrid] Add column Width parameter ([#1902](https://github.com/microsoft/fluentui-blazor/pull/1902))

v4.7.0:
![image](https://github.com/microsoft/fluentui-blazor/assets/86220341/4437f5d2-9543-41c3-9f91-234488b30b20)

v4.6.1:
![image](https://github.com/microsoft/fluentui-blazor/assets/86220341/64c3b5a3-a988-44cc-9fe2-db15920bf9fc)

Tested with:
`<FluentDataGrid Id="weathergrid" Items=@forecasts GridTemplateColumns="1fr 1fr 1fr 2fr" TGridItem=WeatherForecast>`

**Comment by vnbaaij** [comment]
This is why we released v4.7.1 hours later and unlisted 4.7.0 on NuGet. So, please update.

**Reply by michaelc06** [reply]
Thanks! Updating now...
